src/main/java/com/hms_networks/americas/sc/extensions/system/application/package.html

```diff
@@ -0,0 +1,8 @@
+<HTML>
+<BODY>
+Extension package for managing  a Java application running on the Ewon Flexy system.
```

Comment:
Just realized that there is an extra space between 'managing' and 'a'.

Reply:
fixed